crates/re_data_store/src/log_db.rs

```diff
@@ -36,7 +36,12 @@ impl Default for EntityDb {
             tree: crate::EntityTree::root(),
             data_store: re_arrow_store::DataStore::new(
                 InstanceKey::name(),
-                DataStoreConfig::default(),
+                DataStoreConfig {
+                    // NOTE: Empirical testing has shown that 128 is a good balance between sorting
+                    // and binary search costs with the current GC implementation.
```

Comment:
Then maybe make that the `DataStoreConfig::default`?

Reply:
What has the testing be done on btw - huge tensor stuff, or tiny components like `clock`? does it matter?

Reply:
GC is all about finding and sorting buckets; the size of the data itself doesn't impact performance.